contracts/modules/governance/VotePower.sol

```diff
@@ -0,0 +1,64 @@
+// SPDX-License-Identifier: GPL-3.0-only
+
+pragma solidity ^0.8.0;
+
+import "../../interfaces/IGovernance.sol";
+import "../../interfaces/IMemberRoles.sol";
+import "../../interfaces/INXMMaster.sol";
+import "../../interfaces/INXMToken.sol";
+import "../../interfaces/ITokenController.sol";
+
+contract VotePower {
```

Comment:
have you validated balanceOf and totalSupply is enough for snapshot work ?

I assume it is

Reply:
`balanceOf` is the only function that's being called by the `erc20-balance-of` strategy:
https://github.com/snapshot-labs/snapshot-strategies/blob/master/src/strategies/erc20-balance-of/index.ts#L8-L10